install-ci.yml

```diff
@@ -77,6 +77,17 @@
     - role: dd-nodepool
       tags:
         - monitoring
+    - role: logrotate
+      logrotate_configs:
+        - name: nodepool
+          path: /var/log/nodepool/*log
+          options:
+            - compress
+            - missingok
+            - rotate 10
```

Comment:
We definitely don't want to have both rotations enabled at the same time, that would be confusing and half the resulting logs would never get deleted. This PR should also change the nodepool logging template to switch from TimedRotatingFileHandler to WatchedFileHandler (which will correctly handle logrotate changing the underlying inode).